Rivet/Functions/Invoke-Rivet.ps1

```diff
@@ -116,6 +125,47 @@ Found no databases to migrate. This can be a few things:
             return
         }
 
+        if( $DropDatabase )
+        {
+            # Connect to master as we cannot drop a database if we're connected to it
+            Connect-Database -SqlServerName $settings.SqlServerName `
+                                    -Database 'Master' `
+                                    -ConnectionTimeout $settings.ConnectionTimeout
+
+            $databaseString = $settings.Databases | Join-String -Property Name -SingleQuote -Separator ', '
+            $query = "select name from sys.databases where name in ({0})" -f $databaseString
+            $databaseList = Invoke-Query -Query $query
+
+            if( $databaseList )
+            {
+                $confirmDropDatabase = $false
+                if( -not $Force)
+                {
+                    $confirmQuery = "Using the `DropDatabase` switch will drop the database(s) for the current environment. Do you want to proceed?"
+                    $confirmCaption = "Drop the following database(s)? {0}" -f ($databaseList | Join-String -Property Name -Separator ', ')
+                    $confirmDropDatabase = $PSCmdlet.ShouldContinue( $confirmQuery, $confirmCaption )
+                }
+
+                if( $confirmDropDatabase -or $Force )
+                {
+                    foreach( $databaseItem in $databaseList )
+                    {
+                        $query = "DROP DATABASE {0}" -f $databaseItem.Name
+                        Invoke-Query -Query $query
+                    }
+                    return
+                }
+                else

```

Comment:
You can replace all these `else` blocks and `return` statements with a single `return`:

```powershell
                        } # end if( $confirmDropDatabase -or $Force )
                } # end if( $databaseList )
                return
        } # end if( $DropDatabase )

        foreach( $databaseItem in $settings.Databases )